docs/releases/3.2.txt

```diff
@@ -451,6 +451,17 @@ Tests
   comparison against a list of string representations of objects when using the
   default value for the ``transform`` argument.
 
+* Database backends now can mark tests as expected failure by adding them to
```

Comment:
Please put this in a new "Database backends" section instead.

* Third-party database backends can now skip or mark as expected failures tests in Django's test suite using the new DatabaseFeatures.django_test_skips and django_test_expected_failures attributes.

The documentation isn't needed because authors should consult the existing backends to see usage.